queries.sql

```diff
@@ -17,3 +17,65 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3; 
 SELECT * FROM animals WHERE weight_kg Between 10.4 AND 17.3;
+
+BEGIN;
+
+    UPDATE animals
+    SET species = 'unspecified';
+
+    SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+
+SELECT * FROM animals;
```

Comment:
- [ ] Please make sure to use the BEGIN keyword before each transaction.
- [ ] According to the requirements there should be a COMMIT after this transaction and a check if the changes persist after COMMIT